code/datums/emergency_calls/xeno_cultists.dm

```diff
@@ -0,0 +1,37 @@
+//Xeno Cultists
+/datum/emergency_call/xeno_cult
+	name = "Xeno Cultists"
+	mob_max = 10
+	arrival_message = "Incoming Transmission: 'Ia! Ia!'"
+	objectives = "Support the Xenomorphs in any way, up to and including giving your life for them!"
+	probability = 0
+	hostility = TRUE
+	var/max_synths = 1
+	var/synths = 0
+
+/datum/emergency_call/xeno_cult/print_backstory(mob/living/carbon/human/H)
+	to_chat(H, SPAN_BOLD("The xenos are graced the Neroid Sector with their presence! It's time to spread their glory across the stars!"))
+
+/datum/emergency_call/xeno_cult/create_member(datum/mind/M, var/turf/override_spawn_loc)
+	var/turf/spawn_loc = override_spawn_loc ? override_spawn_loc : get_spawn_point()
+
+	if(!istype(spawn_loc))
+		return //Didn't find a useable spawn point.
+
+	var/mob/living/carbon/human/H = new(spawn_loc)
+	M.transfer_to(H, TRUE)
+
+	if(!leader && HAS_FLAG(H.client.prefs.toggles_ert, PLAY_LEADER) && check_timelock(H.client, JOB_SQUAD_LEADER, time_required_for_job))
+		leader = H
+		to_chat(H, SPAN_ROLE_HEADER("You are the leader of this xeno cult! Bring glory to Queen Mother!"))
+		arm_equipment(H, /datum/equipment_preset/other/xeno_cultist/leader, TRUE, TRUE)
+	else if(synths < max_synths && HAS_FLAG(H.client.prefs.toggles_ert, PLAY_SYNTH) && RoleAuthority.roles_whitelist[H.ckey] & WHITELIST_SYNTHETIC)
+		synths++
+		to_chat(H, SPAN_ROLE_HEADER("You are the Xeno Midwife Synth! Tend to the Hive and the captured hosts, make sure the Hive grows!"))
+		arm_equipment(H, /datum/equipment_preset/synth/survivor/midwife, TRUE, TRUE)
```

Comment:
this is a meme preset and admins usually wouldn't want to include it in their events, please remove

Reply:
Switched it to a more serious xeno cultist synthetic gear option.